05/app.js

```diff
@@ -0,0 +1,61 @@
+class Student {
```

Comment:
Rozumiem, że konstruktory to zaszłość, ale w zadaniu byłą mowa o nich to lepiej się trzymać treści zadania (nie chodzi o mnie, ale o potencjalną rekrutację)